### DISCUSSION TITLE
Which integrations would you like next?

### DISCUSSION THREAD

**Discussion by jayantbh** [middlewarehq/middleware#464] [Polls]
Hey everyone! We're taking feedback from the community to decide which integrations should go on our roadmap next. Please feel free to vote for an option below! 👇🏽 

If an integration/category isn't specified, please leave a comment to let us know. 😄 

P.S.: Gitlab isn't listed as an option because it's about to be released, and might be live when you see this discussion. 🚀

P.S.2: Unsure how to make this a multi-choice poll, so, if you'd like to choose more than one, try leaving a comment. 🤷🏽‍♂️

**Comment by abhi-atpl** [comment]
I need integration with Gitea. kindly help

**Comment by jjslagace** [comment]
Hi,
Integration with Azure DevOps would be great.

**Comment by flavio-araujo** [comment]
An webhook for notification, so then I can integrate with anything
